modules/caddyhttp/reverseproxy/fastcgi/caddyfile.go

```diff
@@ -237,7 +237,7 @@ func parsePHPFastCGI(h httpcaddyfile.Helper) ([]httpcaddyfile.ConfigValue, error
 		// route to rewrite to PHP index file
 		rewriteMatcherSet := caddy.ModuleMap{
 			"file": h.JSON(fileserver.MatchFile{
-				TryFiles:  []string{"{http.request.uri.path}", "{http.request.uri.path}/" + indexFile, indexFile},
+				TryFiles:  []string{"{http.request.uri.path}", "{http.request.uri.path}/" + indexFile, "/" + indexFile},
```

Comment:
Am I reading this right, that it'll be, e.g. `/foo/bar/index.php/index.php`?

Reply:
There's a comma there, so no. It's `/index.php`. The change is to just ensure the index file has a `/` prefixed.

But this entire change can probably be thrown away because of the other change in `fastcgi.go` that ensures it at a lower level.

Reply:
Oh, duh -- thanks. I need more sleep. :eyes:

> because of the other change in fastcgi.go that ensures it at a lower level.

You mean this one?

https://github.com/caddyserver/caddy/pull/3570/files#diff-b3eb55bbe16c5572513e0d685dd3b2c9R205-R210

I do think that is the better way to solve it, no need for both I think.

Reply:
Yep. Okay I'll remove the Caddyfile change.